packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -7,9 +7,13 @@ meta:
 
 # Okta Expression Language overview
 
-Expressions allow you to reference, transform, and combine attributes before you store them on a User Profile or before passing them to an application for authentication or provisioning. For example, you might use a custom expression to create a username by stripping `@company.com` from an email address. Or, you might combine the `firstName` and `lastName` attributes into a single `displayName` attribute.
+This document details the features and syntax of Okta Expression Language, which you can use throughout the Okta Admin Console and API for the Okta Classic Engine and Okta Identity Engine.
+
+> **Note:** If you are using the Okta Expression Language for [Global Session Policy and authentication policies](/docs/guides/configure-signon-policy/main/) of the Identity Engine, use the features and syntax of [Okta Expression Language in Okta Identity Engine](/docs/reference/okta-expression-language-in-identity-engine/).
 
-This document details the features and syntax of Okta Expression Language, which you can use throughout the Okta Admin Console and API. Okta updates this document over time as new capabilities are added to the language. Okta Expression Language is based on [SpEL](http://docs.spring.io/spring/docs/3.0.x/reference/expressions.html) and uses a subset of functionalities offered by SpEL.
+Okta updates this document over time as new capabilities are added to the language. Okta Expression Language is based on [SpEL](http://docs.spring.io/spring/docs/3.0.x/reference/expressions.html) and uses a subset of functionalities offered by SpEL.
```

Comment:
`Okta updates this document over time as new capabilities are added to the language`
Hmm I'm not sure if this is appropriate here. Maybe you meant to add it in the other doc? This doc is for the old EL, and I don't think we plan to add new capabilities to the engine. But we might add more functionalities to the newer EL engine.

Reply:
Thanks. Removed. I'm not a huge fan of that sentence in either locations, but will keep it just in OIE for now.